lib/TabsNew/TabsDragLine.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import { string } from 'prop-types';
+import cx from 'classnames';
+
+export function TabsDragLine({ side }) {
+  const className = cx('tab-drag-line absolute top-0 h-full z-10', {
```

Comment:
💚 Can we just add side in like this?
```suggestion
  const className = cx(`tab-drag-line absolute top-0 h-full z-10 ${side}`, {
```